search.html

```diff
@@ -50,6 +50,8 @@
 			r.append($('<div/>', {class: 'search-snippet'}).html(item.htmlSnippet));
 			div.append(r);
 		});
+	} else {
+		div.append($('<p>No results found.</p>'));
```

Comment:
Probably best to make this just `No results.`, to match the corresponding message on the blog search:

<img width="703" alt="screen shot 2017-09-25 at 10 34 28 pm" src="https://user-images.githubusercontent.com/15893490/30839760-d2754d02-a241-11e7-95df-f97e43ca4341.png">

Or update the blog search to match the text here. Either way.